tests/utils.go

```diff
@@ -742,7 +743,7 @@ func AdjustKubeVirtResource() {
 	KubeVirtDefaultConfig = originalKV.Spec.Configuration
 
 	if !flags.ApplyDefaulte2eConfiguration {
-		return
+		return originalKV
```

Comment:
I find it strange when a function changes something and returns the old state - e.g. the caller (usually) has access to the old state before stating its interest in updating it.